pipestat_example/pipeline/count_lines.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+linecount=`wc -l $1 | sed -E 's/^[[:space:]]+//' | cut -f1 -d' '`
+echo "Number of lines: $linecount"
```

Comment:
this pipeline should actually show how to make use of pipestat to report the result.

Reply:
I believe the new changes have addressed these comments.